docs/Development Concepts/Protocols/1.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Waku (v2) and libp2p protocols
```

Comment:
Waku should abstract enough of libp2p so that this is not necessary to understand anything here to use Waku. Hence I'd skip this part section.

Or it can be part of an "advanced" section but I don't see it as necessary either because if one would like to tweak the underneath libp2p object then they can refer to the libp2p doc.

Reply:
gotcha. intention here was to give the readers enough context on the relation/similarities between waku & libp2p

Reply:
Makes sense. Feel free to have a more elaborate section of Waku vs Libp2p.
I think you have the gist of it. On top of my head the angles would be (and we can iterate to make it better):
- Waku as a keyturn solution that includes a number of protocol covering the following domains: privacy preservation, censorship resistance, portability/runs anywhere
- Waku as a service network: user/dev does not have to deploy own infra.